.circleci/upload_to_s3.sh

```diff
@@ -21,5 +21,5 @@ s3cmd \
     --guess-mime-type \
     --acl-public \
     put \
-    ${dir}/* \
-    s3://${bucket}/
+    ${dir} \
```

Comment:
No that can't work. `put` needs files not directories:

From the docs:
```
Put file into bucket
      s3cmd put FILE [FILE...] s3://BUCKET[/PREFIX]
```

So AFAICS you will need to split uploading the `_LATEST-` file and the archive into two steps.